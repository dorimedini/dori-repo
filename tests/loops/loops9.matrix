{
	int f = 3;
	while (f<=24)
	{
		f = f*2;
		print f;
	}
	while (f>=45)
	{
		f = f-1;
		print f;
	}
}

