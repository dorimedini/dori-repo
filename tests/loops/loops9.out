6
12
24
48
47
46
45
44
